Assignment-1/code/index.html

```diff
@@ -1,23 +1,41 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-<meta charset="UTF-8"/>
-<meta name="viewport" content="width=device-width">
-<title>Designing Web App using HTML/CSS/REACT</title>
-<link rel="stylesheet" href="style.css" />
-<script src="https://unpkg.com/react@17/umd/react.development.js"></script>
-<script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
-<!-- Don't use this in production: -->
-<script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width">
+  <title>Designing Web App using HTML/CSS/REACT</title>
+  <link rel="stylesheet" href="style.css" />
+  <script src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+  <!-- Don't use this in production: -->
+  <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
 </head>
+
 <body>
-    <div id="root"></div>
-    <script type="text/babel">
-//React Code here
- ReactDOM.render(<div></div>,document.getElementById('root'));
+  <div id="root"></div>
+  <script type="text/babel">
+    //React Code here
+  var ReactUI=(
+    <div className="container">
+      <div className="toprow">
+        <div className="topleft"></div>
+        <div className="topright"></div>
+      </div>
+      <div className="middlerow">
+        <div className="middlecontent"></div>
+      </div>
+      <div className="bottomrow">
+        <div className="bottomleft"></div>
```

Comment:
We can give a single class name to the divs in bottom row div, it can be much more readable then

Reply:
ok